andela/ah-backend-zeus
# #162163159 Configure project for continuous integration testing with TravisCI

### What does this PR do?
This PR sets up the codebase to automatically run tests on TravisCI each time a commit is pushed to the repository.
#### Description of task to be completed?
* Add TravisCI build file
* Add build status badge to README.md
* Update requirements.txt with dependencies for coverage reporting
#### How should this be manually tested?
* clone this repository
* cd into ah-backend-zeus and checkout the 'ch-integrate-travisci-162163159' branch
* create a new branch from 'ch-integrate-travisci-162163159' and make some commits
* push the work to the origin remote on Github
* click the build status README badge on Github to verify that tests were able to run on TravisCI
#### What are the relevant pivotal tracker stories?
[#162163159](https://www.pivotaltracker.com/n/projects/2227671/stories/162163159)
